packages/core/src/utils/index.ts

```diff
@@ -167,14 +167,6 @@ export function defineReadOnly<T, K extends keyof T>(
   });
 }
 
-export function isBigNumberish(x: any): x is BigNumberish {
```

Comment:
import structure didn't really make sense here. Was going to cause a circular imprort.